lib/postgres_to_redshift.rb

```diff
@@ -8,9 +8,6 @@
 require "postgres_to_redshift/column"
 
 class PostgresToRedshift
-  class << self
-    attr_accessor :source_uri, :target_uri
```

Comment:
`source_uri` and `target_uri` are never used elsewhere besides the specs/unit tests.

Which for right now, I'm not concerned about breaking..